### DISCUSSION TITLE
崩坏：星穹铁道

### DISCUSSION THREAD

**Discussion by jishi148** [share121/inter-knot#2041] [其他]
《崩坏：星穹铁道》是上海米哈游网络科技股份有限公司自研的银河冒险策略游戏 ，是米哈游“崩坏”系列的第五部作品 。游戏于2021年10月27日-2021年11月1日开启“始发测试” ，2022年5月25日-2022年6月15日开启“漫游测试” ，2023年2月10日-2023年3月24日开启“跃迁测试” [32-33]，2023年4月26日正式开启全平台公测 。
游戏中，玩家将登上星穹列车，造访寓居宇宙的万象世界，携手形形色色的旅伴，解救“星核”引发的种种危机，对抗“星神”践踏文明的意图。
剧情设定
这片银河中有名为“星神”的存在，他们造就现实，抹消星辰，在无数“世界”中留下他们的痕迹。
你——一名特殊的旅客，将与继承“开拓”意志的同伴一起，乘坐星穹列车穿越银河，沿着某位“星神”曾经所行之途前进。
你将由此探索新的文明，结识新的伙伴，在无数光怪陆离的“世界”与“世界”之间展开新的冒险。所有你想知道的，都将在群星中找到答案。
那么，准备好开始这段“开拓”之旅了吗？
愿此行，终抵群星。
![562c11dfa9ec8a1363279256d654868fa0ec08fa6a77](https://github.com/user-attachments/assets/1fd78458-339a-47a5-8e7e-2ad9444cc339)

**Comment by Nahida10271027** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/24/2cc5c5050a5aa8dd9be8ff5e8f752e1f_174658425206177111.png" alt="纳西妲-赞" width="75" >（））嗯

**Comment by zhuongxt** [comment]
写的很好，下次不要再写了

**Comment by tomorrowllll** [comment]
你说得对，但
